x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/features/node_allocation/warm_phase.test.ts

```diff
@@ -64,7 +62,8 @@ describe('<EditPolicy /> node allocation in the warm phase', () => {
 
   describe('when using node attributes', () => {
     test('shows spinner for node attributes input when loading', async () => {
-      server.respondImmediately = false;
+      // We don't want the request to resolve immediately.
+      setDelayResponse(true);
```

Comment:
nit: Here is a similar pattern I mentioned for the license reset. A little safer would probably be to use `before` and `after` blocks because if the test fails before reaching line 82, the delayed response won't be reset. But that setup works too.

Reply:
I think this pattern made a lot more sense in the previous case, but for this particular one it will add a lot more boilerplate code. I think I'll leave it as is since its very explicit and limited to this test in particular